### PR TITLE
proofreading spec.tex from Mike

### PR DESCRIPTION
**Pull request by oetiker**
Mike Lee has submitted these changes for spec.tex 

I apologise for this not being via github.
Mostly punctuation changes, only a couple of LaTeX errors (one paragraph position, one footnote position).
My editing comments all begin with %$ to make them easy to find and remove without diff.

In the sans-serif chapter introduction, the two books titles are in an italicised (or emphasised) serif font. That looks intentional, but I raise it just in case (don't normally mix serif with sans, myself).

Trying to work my way to beamer and pgfplots, those being among the very few places I might have anything technical to add.

Mike

### CODE REVIEW

File: src/spec.tex [hunk 1]
```diff
@@ -113,20 +116,28 @@ \section{Indexing}\label{sec:indexing}
...
+
 The \pai*{fancyhdr} package provides a few simple commands that allow you to
+%$$ Consider 'headers and footers' instead of 'header and footer lines'.
 customise the header and footer lines of your document. It defines an
-additional page style \cargv{fancy} and a set of commands to customise it to our liking. By
-default it only adds a line separating the header from the page body.
+additional page style \cargv{fancy} and a set of commands to customise it to
+%$$ Switched 'our' to 'your', as 'our' switches to a collective 'us' from the
+%$$ earlier singular 'you' above; use {'you' & 'your'}, or {'us' & 'our'}.
+%$$ Added comma after 'default'.
```
[the remark this answers is not in the document]
Comment (oetiker) [reply src/spec.tex:136]: yep :) already removed most of them

File: src/spec.tex [hunk 2]
```diff
@@ -99,9 +99,12 @@ \section{Indexing}\label{sec:indexing}
   \csi{printindex}
 \end{lscommand}
 
+%$$ No example of showidx. Might be nice if one could be confined to just this
+%$$ page.
```
Comment (oetiker) [inline src/spec.tex:103]:
```suggestion
```

File: src/spec.tex [hunk 3]
```diff
@@ -113,20 +114,25 @@ \section{Indexing}\label{sec:indexing}
...
 
 \section{Fancy Headers}\label{sec:fancy}
 
 \subsection{Basic commands}
 
+%$$ Not sure if the overall conversational tone of this subsection is in keeping
+%$$ with the bulk of the rest of the document.
+
 The \pai*{fancyhdr} package provides a few simple commands that allow you to
+%$$ Consider 'headers and footers' instead of 'header and footer lines'.
```
Comment (oetiker) [inline src/spec.tex:129]:
```suggestion
```

File: src/spec.tex [hunk 4]
```diff
@@ -147,17 +152,26 @@ \subsection{Basic commands}
...
 combination of three letters:
 \begin{itemize}
-  \item The first specifies whether the location is in the header (\cargv{H}) or in the
-        footer (\cargv{F}).
-  \item The second letter specifies the location within the header\slash{}footer: \cargv{L} for left, \cargv{C} for centre and
-        \cargv{R} for right.
+  \item The first specifies whether the location is in the header (\cargv{H}) or
+        in the footer (\cargv{F}).
+%$$ Changed 'header\slash{}footer' to 'header or footer', although one could
+%$$ go either way here.
```
Comment (oetiker) [inline src/spec.tex:158]:
```suggestion
```

File: src/spec.tex [hunk 5]
```diff
@@ -147,17 +152,24 @@ \subsection{Basic commands}
...
 \begin{itemize}
-  \item The first specifies whether the location is in the header (\cargv{H}) or in the
-        footer (\cargv{F}).
-  \item The second letter specifies the location within the header\slash{}footer: \cargv{L} for left, \cargv{C} for centre and
-        \cargv{R} for right.
+  \item The first specifies whether the location is in the header (\cargv{H}) or
+        in the footer (\cargv{F}).
+  \item The second letter specifies the location within the header or footer:
+%$$ Inserted Oxford comma after centre (this being a list), but opinions may
+%$$ vary on this matter.
```
Comment (oetiker) [inline src/spec.tex:159]:
```suggestion
```

File: src/spec.tex [hunk 6]
```diff
@@ -147,17 +152,22 @@ \subsection{Basic commands}
...
-  \item The second letter specifies the location within the header\slash{}footer: \cargv{L} for left, \cargv{C} for centre and
-        \cargv{R} for right.
+  \item The first specifies whether the location is in the header (\cargv{H}) or
+        in the footer (\cargv{F}).
+  \item The second letter specifies the location within the header or footer:
+        \cargv{L} for left, \cargv{C} for centre, and \cargv{R} for right.
   \item The third letter defines whether the field should be printed on even
-        (\cargv{E}) or odd (\cargv{O}) pages. If the document is not two sided then
-        all pages are treated as odd.
+%$$ Inserted comma after 'sided'.
```
Comment (oetiker) [inline src/spec.tex:160]:
```suggestion
```

File: src/spec.tex [hunk 7]
```diff
@@ -147,17 +152,21 @@ \subsection{Basic commands}
...
   \item The third letter defines whether the field should be printed on even
-        (\cargv{E}) or odd (\cargv{O}) pages. If the document is not two sided then
-        all pages are treated as odd.
+        (\cargv{E}) or odd (\cargv{O}) pages. If the document is not two sided,
+        then all pages are treated as odd.
 \end{itemize}
-So for example the combination \cargv{FCE} identifies the centre part of the
-footer on even pages. If any of the letters is omitted then the identifier
-points toward all positions specifiable by the omitted letter, so for example
+%$$ Changed 'So for example' to 'For example,'
```
Comment (oetiker) [inline src/spec.tex:163]:
```suggestion
```

File: src/spec.tex [hunk 8]
```diff
@@ -147,17 +152,20 @@ \subsection{Basic commands}
...
-        (\cargv{E}) or odd (\cargv{O}) pages. If the document is not two sided then
-        all pages are treated as odd.
+        (\cargv{E}) or odd (\cargv{O}) pages. If the document is not two sided,
+        then all pages are treated as odd.
 \end{itemize}
-So for example the combination \cargv{FCE} identifies the centre part of the
-footer on even pages. If any of the letters is omitted then the identifier
-points toward all positions specifiable by the omitted letter, so for example
+For example, the combination \cargv{FCE} identifies the centre part of the
+%$$ Inserted comma after 'omitted'.
```
Comment (oetiker) [inline src/spec.tex:164]:
```suggestion
```

File: src/spec.tex [hunk 9]
```diff
@@ -147,17 +152,19 @@ \subsection{Basic commands}
...
+        (\cargv{E}) or odd (\cargv{O}) pages. If the document is not two sided,
+        then all pages are treated as odd.
 \end{itemize}
-So for example the combination \cargv{FCE} identifies the centre part of the
-footer on even pages. If any of the letters is omitted then the identifier
-points toward all positions specifiable by the omitted letter, so for example
+For example, the combination \cargv{FCE} identifies the centre part of the
+footer on even pages. If any of the letters is omitted, then the identifier
+%$$ Changed 'letter, so for example' to 'letter. For example,', to break up
+%$$ a long sentence.
```
Comment (oetiker) [inline src/spec.tex:166]:
```suggestion
```

File: src/spec.tex [hunk 10]
```diff
@@ -187,9 +192,14 @@ \subsection{Basic commands}
 \end{document}
 \end{example}
 
-There are two additional commands \csi{fancyhead} and \csi{fancyfoot} that work
-the same way except that they assume \cargv{H} and \cargv{F} respectively
-in their \carg{places} argument unless specified differently.
+%$$ Inserted commas after 'commands' and after '\csi{fancyfoot}'.
```
Comment (oetiker) [inline src/spec.tex:195]:
```suggestion
```

File: src/spec.tex [hunk 11]
```diff
@@ -187,9 +192,13 @@ \subsection{Basic commands}
...
 
-There are two additional commands \csi{fancyhead} and \csi{fancyfoot} that work
-the same way except that they assume \cargv{H} and \cargv{F} respectively
-in their \carg{places} argument unless specified differently.
+There are two additional commands, \csi{fancyhead} and \csi{fancyfoot}, that
+%$$ Changed 'work the' to 'work in the'. Inserted comma after 'way'.
+%$$ Changed 'assum ... respectively' to 'respectively assume ...'.
+work in the same way, except that they respectively assume \cargv{H} and
+%$$ Inserted comma after 'argument'. Changed 'specified differently' to
+%$$ 'otherwise specified'.
```
Comment (oetiker) [inline src/spec.tex:200]:
```suggestion
work in the same way, except that they respectively assume \cargv{H} and
```

File: src/spec.tex [hunk 12]
```diff
@@ -251,13 +261,23 @@ \subsection{Basic commands}
...
-Similarly to \csi{fancyf}, \csi{fancyheadoffset} and \csi{fancyfootoffset} are
-used the same way except they only modify header\slash{}footer.
+%$$ Rewrote this sentence:
+%$ Similarly to \csi{fancyf}, \csi{fancyheadoffset} and \csi{fancyfootoffset} are
+%$ used the same way except they only modify header\slash{}footer.
+The \csi{fancyheadoffset} and \csi{fancyfootoffset} commands are used in the
+same manner as \csi{fancyf}, except that they only modify the header or footer.
+%$$ Actually, that should probably be 'but' instead of 'except that'; there's
+%$$ no 'except' in how they are used; that's the same, the difference is in the
+%$$ result of that usage.
```
Comment (oetiker) [inline src/spec.tex:271]:
```suggestion
The \csi{fancyheadoffset} and \csi{fancyfootoffset} commands are used in the
same manner as \csi{fancyf}, but they only modify the header and footer respectively.
```

File: src/spec.tex [hunk 13]
```diff
@@ -251,13 +261,17 @@ \subsection{Basic commands}
...
+same manner as \csi{fancyf}, but they only modify the header and footer respectively.
 
 \subsection{Contents of the headers}
 
+%$$ Not sure if the overall conversational tone of this subsection is in keeping
+%$$ with the bulk of the rest of the document.
+
 The default footer of the article class contains the current page number. To
-use it inside the fancy header simply use the command \csi{thepage}.
+%$$ Added comma after 'header'. Also 'simply' might be superfluous.
```
Comment (oetiker) [inline src/spec.tex:273]:
```suggestion
```

File: src/spec.tex [hunk 14]
```diff
@@ -269,13 +283,15 @@ \subsection{Contents of the headers}
...
+dedicated to themselves.
 \end{document}%!hide
 \end{example}
 
 It is often useful to have the header and footer contain information based on
-the contents of the page. These are called \enquote{marks} in \LaTeX{}
-terminology. Before we talk about the default ones let's consider how you can
+%$$ Changed 'contents' to 'content' (uncountable noun in this context).
+the content of the page. These are called \enquote{marks} in \LaTeX{}
+%$$ Added comma after 'ones'. Language and abbreviations are informal, here.
```
Comment (oetiker) [inline src/spec.tex:293]:
```suggestion
the content of the page. These are called \enquote{marks} in \LaTeX{}
```

File: src/spec.tex [hunk 15]
```diff
@@ -285,9 +301,12 @@ \subsection{Contents of the headers}
   \csi{lastleftxmark} \\
   \csi{lastrightxmark}
 \end{lscommand}
-The \csi{extramarks} command is used to set the contents of \carg{left} and \carg{right}
+%$$ Changed 'is used to set' to just 'sets'.
+The \csi{extramarks} command sets the contents of \carg{left} and \carg{right}
+%$$ Changed 'Then you can' to 'You can then'.
```
Comment (oetiker) [inline src/spec.tex:306]:
```suggestion
The \csi{extramarks} command sets the contents of \carg{left} and \carg{right}
```

File: src/spec.tex [hunk 16]
```diff
@@ -285,9 +301,12 @@ \subsection{Contents of the headers}
...
 marks. Then you can access these marks inside the headers by using the appropriate
 command. The \texttt{first}- commands refer to the first mark occurring on the
+%$$ Added 'commands' after \texttt{last}-, because still needs to be a plural.
```
Comment (oetiker) [inline src/spec.tex:309]:
```suggestion
```

File: src/spec.tex [hunk 17]
```diff
@@ -313,7 +332,8 @@ \subsection{Contents of the headers}
 \end{example}
 
 Let us now look at the default marks defined by \LaTeX{}. After loading
-\pai{extramarks} these may be set and accessed similarly:
+%$$ Added comma after '\pai{extramarks}'.
```
Comment (oetiker) [inline src/spec.tex:335]:
```suggestion
```

File: src/spec.tex [hunk 18]
```diff
@@ -322,10 +342,16 @@ \subsection{Contents of the headers}
...
-For example in the article class, \carg{left} is set by the \csi{section}
+%$$ Long sentence. Changed 'them, so' to 'them. Hence,'.
+%$$ Added a comma after 'careful'.
+%$$ Furthermore, does 'them' mean 'these' (as opposed to the extra marks)?
+automatically fill them. Hence, if you are not careful, you may lose your
+%$$ Removed whitespace between 'content.' and the footnote.
+content.\footnote{You may prevent this by setting the pagestyle to
+\cargv{myheadings} and then redefining it to use it with \pai{fancyhdr} as
+described later.}
+%$$ Added comma after 'for example'.
```
Comment (oetiker) [inline src/spec.tex:353]:
```suggestion
automatically fill them. Hence, if you are not careful, you may lose your
content.\footnote{You may prevent this by setting the pagestyle to
\cargv{myheadings} and then redefining it to use it with \pai{fancyhdr} as
described later.}
```

File: src/spec.tex [hunk 19]
```diff
@@ -346,12 +372,15 @@ \subsection{Sub}
 \end{document}%!hide
 \end{example}
 Note that \csi{firstrightmark} is empty in the above example. This is caused by
-the fact that \csi{section} commands sets both marks, leaving the right one
+%$$ Changed 'set' to 'sets', to avoid the double plural with 'commands'.
```
Comment (oetiker) [inline src/spec.tex:375]:
```suggestion
```

File: src/spec.tex [hunk 20]
```diff
@@ -346,12 +372,15 @@ \subsection{Sub}
...
+the fact that \csi{section} commands set both marks, leaving the right one
 empty.
 
-You may have noticed that section titles are typeset in uppercase when provided by
-marks commands. To disable this use the \csi{nouppercase} command inside the \csi{fancyhf}
-command.
+You may have noticed that section titles are typeset in uppercase when provided
+%$$ Consider "`marks'" or "``marks''" or "mark" or -\texttt{mark} instead of
+%$$ "marks" (in 'marks commands', on the next line).
+by marks commands. To disable this use the \csi{nouppercase} command inside the
```
Comment (oetiker) [inline src/spec.tex:382]:
```suggestion
by -\texttt{mark} commands. To disable this use the \csi{nouppercase} command inside the
```

File: src/spec.tex [hunk 21]
```diff
@@ -368,9 +397,12 @@ \section{Today}
 \end{document}%!hide
 \end{example}
 
+
 \subsection{Advanced commands}
 
-If you want even more control over the section titles you may redefine the
+%$$ Changed 'the section titles' to 'section titles,'.
+%$$ Consider 'can' instead of 'may', the latter suggests permission to do so.
+If you want even more control over section titles, you may redefine the
```
Comment (oetiker) [inline src/spec.tex:405]:
```suggestion
If you want even more control over section titles, you can redefine the
```

File: src/spec.tex [hunk 22]
```diff
@@ -545,6 +605,10 @@ \section{Section}
 You may be disappointed.
 \end{document}
 \end{example}
+%$$ I don't think I understand the above example.
+%$$ The connection between 'paradox' on the left, and 'disappointed' on the 
+%$$ right isn't obvious. Did I miss something?
```
[the remark this answers is not in the document]
Comment (rmrstar) [reply src/spec.tex:567]: Might just have been me being slow. Usually throughout the document,, the code on the left produces the render on the right. So when I'm looking at the PDF in isolation, I'm scratching my head over how did that code (the 'paradox' line) produce the 'disappointed' line on the other side. What settings did I miss that overrode or hid that line? Was there a header setting somewhere to add the 'disappointed' line? Upon reading the .tex source, it's obvious; there's no connection between the sides, besides the 'My style' header. I get the joke, but if you don't have the source at hand...

**Pull request merged**